fit_curve.json

```diff
@@ -85,13 +82,10 @@
         }
     ],
     "returns": {
-        "description": "An array with the optimal values for the parameters.",
+        "description": "A data cube with the optimal values for the parameters.",
```

Comment:
parameters is not a datacube, it's just a list I think! These are the initial guesses for the fitting function!